instagram/src/components/PostContainer/PostList.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Post from './Post';
+import Likes from './Likes';
+import CommentList from '../CommentSection/CommentList';
+
+const PostList = props => {
+  return (
+    <div>
+      {props.data.map((post) => {
+        return (
+          <div className='postContainer'>
+            <Post userData={post}/>
+            <Likes likes={post.likes}/>
+            <CommentList comments={post.comments}/>
+            <p className='timeStamp'>{post.timestamp}</p>
+            <input className='addComment' type='text' placeholder='Add a comment...' />
+          </div>
+        )})}
+    </div>
+  )
+}
+
+PostList.propTypes = {
+ timestamp: PropTypes.string, 
```

Comment:
_Technically_ timestamp isn't a prop that is being passed directly to PostList, only data is it looks like, so define proptypes for that

Reply:
I don't fully understand the second part.

Reply:
You should do something like

```
PostList.propTypes = {
 data: PropTypes.arrayOf(PropTypes.object),
}
```